xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/servletEngine/ServletContainerExecutor.java

```diff
@@ -170,6 +173,77 @@ private void startContainer() throws Exception
         start(this.servletContainer, this.testConfiguration);
     }
 
+    private void throwExceptionForServletEngineNotYetSupported(TestConfiguration testConfiguration)
+    {
+        throw new RuntimeException(String.format("Servlet engine [%s] is not yet supported!",
+            testConfiguration.getServletEngine()));
+    }
+
+    private GenericContainer getServletContainer(TestConfiguration testConfiguration)
+    {
+        final String baseImageName;
+
+        final GenericContainer container;
+
+        switch (testConfiguration.getServletEngine()) {
+            case TOMCAT:
+                baseImageName = String.format("tomcat:%s", testConfiguration.getServletEngineTag() != null
+                    ? testConfiguration.getServletEngineTag() : LATEST);
+                break;
+
+            case JETTY:
+                baseImageName = String.format("jetty:%s", testConfiguration.getServletEngineTag() != null
+                    ? testConfiguration.getServletEngineTag() : LATEST);
+                break;
+
+            case WILDFLY:
+                baseImageName = String.format("jboss/wildfly:%s", testConfiguration.getServletEngineTag() != null
+                    ? testConfiguration.getServletEngineTag() : LATEST);
+                break;
+
+            default:
+                this.throwExceptionForServletEngineNotYetSupported(testConfiguration);
+                baseImageName = null;
+        }
+
+        // we want to use libreoffice so we need a custom image
```

Comment:
We don't want to use LO. Only if isOffice() return trues.

Reply:
Also could you please start all your sentences by an uppercase? :) I've noticed that you always start with a lowercase...